package.json

```diff
@@ -1,98 +1,82 @@
 {
   "name": "availity-starter-react",
+  "version": "0.1.0",
   "private": true,
-  "version": "5.3.4",
   "description": "React project using @availity/workflow",
-  "main": "index.js",
+  "keywords": [
+    "react",
+    "availity",
+    "workflow"
+  ],
+  "homepage": "https://github.com/availity/availity-workflow#readme",
+  "license": "MIT",
+  "main": "project/app/index.js",
   "scripts": {
-    "start": "av start",
     "av": "av",
-    "test": "av test",
-    "test:watch": "av test --watch",
-    "test:integration": "av test",
     "about": "av about",
-    "lint": "av lint",
-    "help": "av help",
-    "upgrade:workflow": "./node_modules/.bin/upgrade-workflow",
-    "profile": "av profile",
-    "profile:production": "cross-env NODE_ENV=production av profile",
     "build": "av build",
     "build:production": "cross-env NODE_ENV=production av build",
     "build:staging": "cross-env NODE_ENV=staging av build",
-    "release": "av release",
-    "prettier": "prettier --write '**/*.{ts,tsx,js,scss}'",
+    "help": "av help",
+    "lint": "av lint",
+    "format": "prettier --write '**/*.{ts,tsx,js,scss}'",
+    "format:check": "prettier --check '**/*.{ts,tsx,js,scss}'",
     "production": "cross-env NODE_ENV=production av release",
-    "staging": "cross-env NODE_ENV=staging av release"
+    "profile": "av profile",
+    "profile:production": "cross-env NODE_ENV=production av profile",
+    "release": "av release",
+    "staging": "cross-env NODE_ENV=staging av release",
+    "start": "av start",
+    "test": "av test",
+    "test:integration": "av test",
+    "test:watch": "av test --watch",
+    "upgrade:workflow": "./node_modules/.bin/upgrade-workflow"
+  },
+  "resolutions": {
+    "core-js": "3.19.3"
   },
-  "keywords": [
-    "react",
-    "availity",
-    "workflow"
-  ],
-  "license": "MIT",
-  "homepage": "https://github.com/availity/availity-workflow#readme",
   "dependencies": {
-    "@availity/api-axios": "^6.0.5",
-    "@availity/form": "^1.2.2",
-    "@availity/icon": "^0.10.1",
-    "@availity/link": "^2.4.1",
-    "@availity/page-header": "^11.1.10",
-    "@availity/spaces": "^5.1.1",
-    "@availity/yup": "^4.1.0",
+    "@availity/api-axios": "^6.0.6",
+    "@availity/form": "^1.2.3",
+    "@availity/icon": "^0.10.2",
+    "@availity/link": "^2.4.2",
+    "@availity/page-header": "^11.1.13",
+    "@availity/spaces": "^5.1.2",
+    "@availity/yup": "^4.2.0",
     "availity-uikit": "^4.1.5",
-    "axios": "^0.24.0",
+    "axios": "^0.25.0",
     "browserslist": "4.19.1",
     "caniuse-lite": "1.0.30001286",
     "classnames": "^2.3.1",
+    "core-js": "3.19.3",
```

Comment:
is `core-js` needed as a direct dependency? It will be included with some sdk-js packages and `@availity/workflow`, the `resolution` field should still work on nested dependencies if you are pinning to avoid a bug

Reply:
You're right. I should only use the `resolution` field